### DISCUSSION TITLE
Contribution preferences

### DISCUSSION THREAD

**Discussion by jonathankretzmer** [rainepretorius/olarm-ha-integration#35] [General]
Hey,

I just wanted to get an idea of how you would like contributions to be done going forward. Feature or release branches, hotfix branches, etc.

Would like to hear how you would prefer for this to be managed :)

**Comment by AlduCor** [comment]
I also have a PR ready, but would like to wait to hear what the preferences are for contributions.

**Comment by rainepretorius** [comment]
Hi,
When your changes are ready to commit to the master / main production branch. Just create a pull request. I will then go to your pull request and copy it to the development server I use to run certain tests. If all tests pass, your changes will be brought to the master branch.
